CODE_REVIEW_CHECKLIST.md

```diff
@@ -69,3 +71,10 @@ may have.
 
 - [ ] Will this change work on all of our supported platforms and
       browsers?
+
+- [ ] Do we expect significant changes in performance due to the
```

Comment:
I think it's probably fine as is-- My concern is that this is an easy one to sort of gloss over if not asked to list or discuss "possible ramifications", but I don't think the overhead of requesting an answer is worthwhile, here (unless we see a trend of PRs that have no consideration of performance).